darwin/PlatformHelpers.c

```diff
@@ -0,0 +1,124 @@
+/*
+htop - DarwinProcessList.c
+(C) 2014 Hisham H. Muhammad, 2021 Alexander Momchilov
+Released under the GNU GPLv2, see the COPYING file
+in the source distribution for its full text.
+*/
+
+#include "darwin/PlatformHelpers.h"
+
+#include <errno.h>
+#include <unistd.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/sysctl.h>
+
+#include "CRT.h"
+
+#ifdef HAVE_MACH_MACH_TIME_H
+#include <mach/mach_time.h>
+#endif
+
```

Comment:
Double blank line here …

Reply:
🧐 There's only a single blank line?

Reply:
It should be two blank lines …
Cf. [here](https://github.com/htop-dev/htop/blob/main/docs/styleguide.md#import-and-use-of-headers).

Reply:
Oh I see, you're request there be two blank lines. Got it. I thought you were complaining there were excess lines.

Looks like `DarwinProcess.h` already violates that style rule: https://github.com/htop-dev/htop/blob/main/darwin/DarwinProcess.h#L14 I'll fix it.